webhookGithubApp/index.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const axios = require('axios');
+const azureWrapper = require('../util/azureWrapper');
+const { createAppAuth } = require('@octokit/auth-token');
+const config = require('../.config');
+const mongoose = require('mongoose');
+
+let conn = null;
+const subscriberSchema = new mongoose.Schema({
+  email: { type: String, required: true },
+  githubUsername: { type: String, required: true },
+  githubUserId: { type: String },
+  githubOrganization: { type: String },
+  githubOrganizationId: { type: String }
+});
+
+module.exports = azureWrapper(async function webhookGithubApp(context, req) {
+  let Subscriber;
+  if (conn == null) {
+    conn = mongoose.createConnection(config.uri);
+    await conn.asPromise();
+  }
+
+  Subscriber = conn.model('Subscriber', subscriberSchema, 'Subscriber');
+
+  const { installation, sender } = req.body;
+  const auth = createAppAuth({
+    id: ,
+    privateKey: ,
+    installationId: installation.id,
+    clientId: ,
+    clientSecret: 
+  });
+  const { token } = await auth({ type: 'installation' });
+
+  if(installation.account.type == 'Organization') {
```

Comment:
Make this a `if(installation.account.type !== 'Organization') { return; }` and move it up to before `createAppAuth()` to we can avoid extra indentation and not unnecessarily create app auth if we don't need it.